build.yaml

```diff
@@ -0,0 +1,112 @@
+                CPP_DEV_DRIVER_LOCATION=http://downloads.datastax.com/cpp-driver/$DISTRO/$RELEASE/cassandra/v$CPP_DRIVER_VERSION/cassandra-cpp-driver-devel-$CPP_DRIVER_VERSION-${CPP_DRIVER_SUBVERSION}.el${RELEASE}.centos.x86_64.rpm
+                CPP_DRIVER=cassandra-cpp-driver-$CPP_DRIVER_VERSION-${CPP_DRIVER_SUBVERSION}.el${RELEASE}.centos.x86_64.rpm
+                CPP_DEV_DRIVER=cassandra-cpp-driver-devel-$CPP_DRIVER_VERSION-${CPP_DRIVER_SUBVERSION}.el${RELEASE}.centos.x86_64.rpm
+              elif [ "${RELEASE}" = "6" ]; then
+                CPP_DRIVER_LOCATION=http://downloads.datastax.com/cpp-driver/$DISTRO/$RELEASE/cassandra/v$CPP_DRIVER_VERSION/cassandra-cpp-driver-$CPP_DRIVER_VERSION-${CPP_DRIVER_SUBVERSION}.el${RELEASE}.x86_64.rpm
+                CPP_DEV_DRIVER_LOCATION=http://downloads.datastax.com/cpp-driver/$DISTRO/$RELEASE/cassandra/v$CPP_DRIVER_VERSION/cassandra-cpp-driver-devel-$CPP_DRIVER_VERSION-${CPP_DRIVER_SUBVERSION}.el${RELEASE}.x86_64.rpm
+                CPP_DRIVER=cassandra-cpp-driver-$CPP_DRIVER_VERSION-${CPP_DRIVER_SUBVERSION}.el${RELEASE}.x86_64.rpm
+                CPP_DEV_DRIVER=cassandra-cpp-driver-devel-$CPP_DRIVER_VERSION-${CPP_DRIVER_SUBVERSION}.el${RELEASE}.x86_64.rpm
+              else
+                printf "Unsupported Centos Version: %s\n" ${RELEASE}
+                exit 1
+              fi
+            else
+              printf "Unsupported OS: %s\n" ${OS_VERSION}
+              exit 1
+            fi
+
+            pushd ext/packaging            
+            mkdir packages
+
+            # Build libuv, install, and copy to "packages"
+            echo "Building libuv Packages [${LIBUV_VERSION}] ...\n"
+            git clone --depth 1 https://github.com/mpenick/libuv-packaging.git libuv-packaging
+            
+            pushd libuv-packaging
+            ./build_${PACKAGE_TYPE}.sh ${LIBUV_VERSION}
+            popd
+            find libuv-packaging/build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \;
+            sudo ${PACKAGE_INSTALL} packages/libuv*.${PACKAGE_TYPE}
+
+            # Download and Install cpp core drivers
+            wget $CPP_DRIVER_LOCATION $CPP_DEV_DRIVER_LOCATION
+            sudo ${PACKAGE_INSTALL} $CPP_DRIVER $CPP_DEV_DRIVER
+
+            # Build, package and upload DataStax PHP driver
+            echo "Building DataStax PHP driver ..."
+            for PHP_VERSION in "${PHP_VERSIONS[@]}"
+                do
+                  sudo ${PACKAGE_UTILITY} install -y ${PHP_VERSION}
+                  ./build_${PACKAGE_TYPE}.sh
+                  find build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \; 
+                  rm -rf build
+                  PHP_TOKENS=($(echo ${PHP_VERSION} | tr "-" " "))
+                  sudo ${PACKAGE_UTILITY} remove -y ${PHP_TOKENS[0]}-common
```

Comment:
Use `${PACKAGE_REMOVE}` which would be `PACKAGE_REMOVE=${PACKAGE_UTILITY} remove -y`.